### PR TITLE
log YAML output on create, update and delete

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

File: ocp_resources/resource.py [hunk 1]
```diff
@@ -640,6 +640,7 @@ def create(self, body=None, wait=False):
             data.update(body)
 
         LOGGER.info(f"Posting {data}")
```
Comment (yossisegev) [inline ocp_resources/resource.py:642]: I think that this one is redundant now.

**Review by yossisegev — CHANGES_REQUESTED** [review]

**Review by yossisegev — COMMENTED** [review]
-

File: ocp_resources/logger.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import logging
+import os
+
+from colorlog import ColoredFormatter
+
+
+def get_logger(name):
+    log_level = os.environ.get("OPW_LOG_LEVEL", "INFO")
```
Comment (rnetser) [inline ocp_resources/logger.py:8]: 1. Can you please update the README?
2. What does OPW stand for?

**Review by rnetser — CHANGES_REQUESTED** [review]
"ceate" -> "create" in the title and description

File: ocp_resources/logger.py (hunk 2 above)
Comment (yossisegev) [reply ocp_resources/logger.py:8]: > What does OPW stand for?

My guess is "Openshift Python Wrapper".

Comment (myakove) [reply ocp_resources/logger.py:8]: right, please suggest a better name.

Comment (rnetser) [reply ocp_resources/logger.py:8]: why not OCP_PYTHON_WRAPPER_LOG_LEVEL ? long but self explanatory.

**Review by rnetser — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**